site/reference/fn/eval/README.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "`eval`"
+linkTitle: "eval"
+type: docs
+description: >
+  Execute function on resources
+---
+
+<!--mdtogo:Short
+    Execute function on resources
+-->
+
+`eval` executes a function on resources in a directory. Functions are packaged
+as container images or binary executables.
+
+If the function fails (i.e. exits with non-zero status code), `eval` will
+abort and the local filesystem is left intact.
+
+Refer to the [Imperative Function Execution] for detailed overview.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```shell
+kpt fn eval [DIR|-] [flags] [-- fn-args]
+```
+
+#### Args
+
+```shell
+DIR|-:
+  Path to the local directory containing resources. Defaults to the current
+  working directory. Using '-' as the directory path will cause `eval` to
+  read resources from `stdin` and write the output to `stdout`.
+```
+
+#### Flags
+
+```shell
+--as-current-user:
+  Use the `uid` and `gid` of the kpt process for container function execution.
+  Container functions are executed as `nobody` user which has very limited
+  privileges. You may want to use this flag to run higher privilege operations
+  such as mounting the local filesystem.
+
+--dry-run:
+  If enabled, the resources are not written to local filesystem, instead they
+  are written to stdout. By defaults it is disabled.
+  
+--env, e:
+  List of local environment variables to be exported to the container function.
+  By default, none of local environment variables are made available to the
+  container running the function. The value can be in `key=value` format or only
+  the key of an already exported environment variable.
+
+--exec-path:
+  Path to the local executable binary to execute as a function. `eval` executes
```

Comment:
Expand this to say it's for local testing purposes.

Reply:
Done.